packages/core/src/index.ts

```diff
@@ -16,9 +16,10 @@ export * from './types'
 const getCSS = props => {
   if (!props.sx && !props.css) return undefined
   return theme => {
-    const styles = css(props.sx)(theme)
+    const sx = Array.isArray(props.sx) ? props.sx : [props.sx]
+    const styles = sx.map(s => css(s)(theme))
```

Comment:
@jxnblk Do you have a preferred method for recursively flattening arrays? I realized after I opened the PR that Emotion recursively flattens arrays of styles and this doesn't account for that scenario.

Reply:
There are merge utilities in this file (see line 66)—is that what you’re looking for?

Reply:
@lachlanjc Thanks for the suggestion. Looks like the merge utility didn't have what i was looking for. I update a PR to do the flattening though.